cdap-ui/package.json

```diff
@@ -139,6 +139,8 @@
     "@material-ui/core": "4.0.1",
     "@material-ui/icons": "4.0.1",
     "ace-builds": "1.4.4",
+    "apollo-server": "^2.6.7",
+    "apollo-server-express": "^2.6.7",
```

Comment:
remove the `^` from the new dependencies added, and rerun `yarn install` so that the `yarn.lock` file is updated

Reply:
Done